src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -394,16 +400,18 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 contactsChangedLastName.add(c);
             }
 
-
-            //If Contact Name or Exclude_from_Household_Name__c value has changed
-            if ((conLastNameChanged ||
-                 conFirstNameChanged ||
-                 conExcludeFromHHNameChanged) &&
-                 numOfContactsLeft != 0 &&
-                 conAccountIdNotNull &&
-                 checkHouseholdNaming(c.Account.RecordTypeId))
-            {
-                accountIdToRename.add(c.AccountId);
+            //If Contact Name or Exclude_from_Household_Name__c or Deceased value has changed
+            //and numOfContactsLeft is not 0, rename the Account record
+            if (conAccountIdNotNull && checkHouseholdNaming(c.Account.RecordTypeId)) {
+                if (((conLastNameChanged ||
+                      conFirstNameChanged ||
+                      conExcludeFromHHNameChanged ||
+                      conDeceasedHasChanged) &&
+                 	  numOfContactsLeft != 0)
+                   )
+                {
+                	accountIdToRename.add(c.AccountId);
```

Comment:
NIT: since this is a list, could we call it "accountIdsToRename" (notice the "s") to make it plural. It wouldn't let me comment on it from line 342 as it might be legacy but I keep reading it and find myself thinking there's only one..